components/Base/Button.vue

```diff
@@ -48,7 +48,7 @@ const buttonLookVariants = ['fill', 'outline'];
 
 const componentType = computed(() => {
   if (props.href) return 'a';
-  if (props.to) return 'nuxt-link';
```

Comment:
For some reason this didn't generate `a` tag but `nuxt-link` in our html result. `router-link` works fine though.